backend/src/test/java/reviewme/review/domain/ReviewTest.java

```diff
@@ -25,4 +35,62 @@ class ReviewTest {
         assertThatThrownBy(() -> new Review(member, member, null, keywords, createdAt))
                 .isInstanceOf(IllegalReviewerException.class);
     }
+
+    @Test
+    void 마감_기한이_지난_그룹에_리뷰를_등록할_수_없다() {
+        // given
+        Member reviewer = 회원_산초.create();
+        Member reviewee = 회원_아루.create();
+        ReviewerGroup reviewerGroup = 데드라인_지난_그룹.create(reviewee, List.of(reviewer.getGithubId()));
+        LocalDateTime createdAt = LocalDateTime.now();
+        List<Keyword> keywords = List.of();
+
+        // when, then
+        assertThatThrownBy(() -> new Review(reviewer, reviewee, reviewerGroup, keywords, createdAt))
+                .isInstanceOf(DeadlineExpiredException.class);
+    }
+
+    @Test
+    void 하나의_리뷰_그룹에_중복으로_리뷰를_등록할_수_없다() {
+        // given
+        Member reviewer = 회원_산초.create();
+        Member reviewee = 회원_아루.create();
+        ReviewerGroup reviewerGroup = 데드라인_남은_그룹.create(reviewee, List.of(reviewer.getGithubId()));
+        new Review(reviewer, reviewee, reviewerGroup, List.of(), LocalDateTime.now());
+        LocalDateTime createdAt = LocalDateTime.now();
+        List<Keyword> keywords = List.of();
+
+        // when, then
+        assertThatThrownBy(() -> new Review(reviewer, reviewee, reviewerGroup, keywords, createdAt))
+                .isInstanceOf(ReviewAlreadySubmittedException.class);
+    }
+
+    @Test
+    void 리뷰어로_등록되지_않은_회원은_리뷰를_등록할_수_없다() {
+        // given
+        Member reviewer = 회원_산초.create();
+        Member reviewee = 회원_아루.create();
+        ReviewerGroup reviewerGroup = 데드라인_남은_그룹.create(reviewee, List.of(new GithubId(3)));
```

Comment:
이 부분은 fixture를 사용하지 않고 member를 직접 생성하는 건 어떨까요?
리뷰어 그룹인 사용자 아이디는 3이나, 리뷰어가 아닌 아이디가 1인 사용자가(산초)가 리뷰를 하려는 상황이라는 게 드러나면 좋을 것 같아서요! 
```suggestion
        Member reviewer = 회원_산초(1001L, "산초",  new GithubId(1));
        Member reviewee = 회원_아루.create();
        ReviewerGroup reviewerGroup = 데드라인_남은_그룹.create(reviewee, List.of(new GithubId(3)));
```

Reply:
테스트에서 확인되는 게 좋겠네요. 수정합니다 !